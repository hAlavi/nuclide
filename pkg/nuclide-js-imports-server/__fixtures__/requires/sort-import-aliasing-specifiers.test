
import type {A as D, B as C} from 'AB';
import type {Foo} from 'Foo';

import {M as Q, N as O} from 'AB';

function foo(a: C, b: D, foo: Foo) {
  return [Q, O];
}

// @nolint
