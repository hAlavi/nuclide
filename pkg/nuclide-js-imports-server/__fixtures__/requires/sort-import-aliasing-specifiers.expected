import type {B as C, A as D} from 'AB';
import type {Foo} from 'Foo';

import {N as O, M as Q} from 'AB';

function foo(a: C, b: D, foo: Foo) {
  return [Q, O];
}

// @nolint
